e2e/test/scenarios/navbar.cy.spec.js

```diff
@@ -0,0 +1,112 @@
+import {
+  describeEE,
+  setTokenFeatures,
+  popover,
+  visitDashboard,
+  restore,
+} from "e2e/support/helpers";
+
+describe("scenarios > navigation > navbar", () => {
+  describe("OSS", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsNormalUser();
+    });
+
+    it("should be open after logging in", () => {
+      cy.visit("/");
+      cy.findByTestId("main-navbar-root").should("be.visible");
+    });
+
+    it("state should preserve when clicking the mb logo", () => {
+      cy.visit("/collection/root");
+      cy.findByTestId("main-navbar-root").should("be.visible");
+      cy.findByTestId("main-logo-link").click();
+      cy.findByTestId("main-navbar-root").should("be.visible");
+      visitDashboard(8);
```

Comment:
anyone know where I can import constants for things like this. i make a few magical references in this file that i'd like to fix...